content/blog/tidyclust-0-1-0/index.Rmd

```diff
@@ -0,0 +1,150 @@
+---
+output: hugodown::hugo_document
+
+slug: tidyclust-0-1-0
+title: tidyclust is on CRAN
+date: 2022-12-06
+author: Emil Hvitfeldt
+description: >
+    Tidyclust is on CRAN. tidyclust provides a common interface for specifying 
+    clustering models, in the same style as parsnip.
+
+photo:
+  url: https://unsplash.com/photos/4Xy08NbMBLM
+  author: Ankush Minda
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, tidyclust]
+---
+
+<!--
+TODO:
+* [X] Look over / edit the post's title in the yaml
+* [X] Edit (or delete) the description; note this appears in the Twitter card
+* [X] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [X] Find photo & update yaml metadata
+* [X] Create `thumbnail-sq.jpg`; height and width should be equal
+* [X] Create `thumbnail-wd.jpg`; width should be >5x height
+* [X] `hugodown::use_tidy_thumbnails()`
+* [X] Add intro sentence, e.g. the standard tagline for the package
+* [X] `usethis::use_tidy_thanks()`
+-->
+
+We're very pleased to announce the release of [tidyclust](https://tidyclust.tidymodels.org/) 0.1.0. tidyclust is the tidymodels extension to fit and use clustering models.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("{package}")
+```
+
+This blog post will introduce tidyclust, how to use it with the rest of tidymodels, and how we can interact and evaluate the fitted cluster models.
```

Comment:
```suggestion
This blog post will introduce tidyclust, how to use it with the rest of tidymodels, and how we can interact and evaluate the fitted clustering models.
```

You mostly use "clustering" elsewhere in this post—if these refer to the same thing, I'd choose one phrasing or the other. :)